api/src/utils/extendedSingleNodeClient.ts

```diff
@@ -1,11 +1,19 @@
+/* eslint-disable max-len */
+/* eslint-disable no-unused-vars */
```

Comment:
Not sure why we are disabling this rule? Shouldn't really have any unused variables here. 

Also, isn't it redundant anyways because we also have:

```
// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Reply:
Both rules need to be disabled or the build fails. Can add them to one line though:`// eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars`

Reason for disabling is the deconstruction of the request object on line 13. 
extendedSingleNodeClient  uses address property and optional page_size or state properties to create query params. Network property is not used here but is a part of request because it is used in transactionhistory/get.ts to create networkConfig.